### ISSUE TITLE
Question about original plane

### ISSUE THREAD

**Issue by Dou-Yiming** [facebookresearch/3D-Vision-and-Touch#7]
Hi, I have a further question about the model.
During the initialization of the model, a plane is defined by: 

`width = .0218 - 0.00539
y_z = torch.arange(dim).cuda().view(dim, 1).expand(dim, dim).float()
y_z = torch.stack((y_z, y_z.permute(1, 0))).permute(1, 2, 0)
plane = torch.cat((torch.zeros(dim, dim, 1).cuda(), y_z), dim=-1)
self.orig_plane = (plane / float(dim) - .5) * width`

Based on my understanding, the prediction of the model is based on the deformation of this plane, may I ask what's the meaning of "width"? It is a parameter related to your simulator? 
Thanks so much for your help!

**Comment by EdwardSmith1884** [comment]
oh so width describes the the real width of the sensor in simulation. This way the dimension of the plane being deformed matches the touch sensor.

**Comment by Dou-Yiming** [comment]
> oh so width describes the the real width of the sensor in simulation. This way the dimension of the plane being deformed matches the touch sensor.

Yes, I've figured out what it means, thanks so much!
